main.css

```diff
@@ -46,109 +48,123 @@ h1 {
 	line-height: 200%;
 }
 
-.menu > li {
+#navigation .menu > li {

```

Comment:
@nattix to jest dobrze. Jest też inna metoda. Jeżeli po prostu napiszesz: `#navigation .menu li`. Css zrozumie, ze chodzi Ci o dziecko czyli `li`. Wiec nie musisz wówczas używać operatora `>`